.github/workflows/build.yml

```diff
@@ -77,6 +67,13 @@ jobs:
           name: packages
           path: ./*/build/distributions/*.zip
 
+      - name: Upload coverage to Codecov
+        # We need coverage data from only one the builds
+        if: ${{ matrix.os }} == ubuntu-latest
```

Comment:
@maciejwalkowiak I just copied from `Archive packages` but this seems to not work, its executed on mac-os too

Reply:
@bruno-garcia set this up for Archive Packages. Perhaps take a look here: https://github.community/t/what-is-the-correct-if-condition-syntax-for-checking-matrix-os-version/16221/3

Reply:
`runner.os == 'Linux'` instead works.